x11-wm/mutter/mutter-42.1.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
-PYTHON_COMPAT=( python3_{8..10} )
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
```

Comment:
Just to check: you ran test suites for all of these too to make sure they pass, if adding 3.11? If you did, then great!

Reply:
Not all (or admittedly, most) yet, but I also don't expect this PR to be merged, just want to catch any immediate issues in review before my branch gets too far behind.

As I break it up over the next few days I'll properly test packages and fix or revert anything that doesn't work.